regression/scripts/ducted_fan/serial_hybrid_ducted_fan_network.py

```diff
@@ -0,0 +1,276 @@
+    #compute engine areas
+    ducted_fan.areas.wetted      = 1.1*np.pi*ducted_fan.nacelle_diameter*ducted_fan.engine_length
+    
+    # working fluid
+    ducted_fan.working_fluid = SUAVE.Attributes.Gases.Air()
+
+    # ------------------------------------------------------------------
+    #   Component 1 - Ram
+    
+    # to convert freestream static to stagnation quantities
+    # instantiate
+    ram         = SUAVE.Components.Energy.Converters.Ram()
+    ram.tag     = 'ram'
+    
+    # add to the network
+    ducted_fan.append(ram)
+
+    # ------------------------------------------------------------------
+    #  Component 2 - Inlet Nozzle
+    
+    # instantiate
+    inlet_nozzle        = SUAVE.Components.Energy.Converters.Compression_Nozzle()
+    inlet_nozzle.tag    = 'inlet_nozzle'
+    
+    # setup
+    inlet_nozzle.polytropic_efficiency = 0.98
+    inlet_nozzle.pressure_ratio        = 0.98
+    
+    # add to network
+    ducted_fan.append(inlet_nozzle)
+
+    # ------------------------------------------------------------------
+    #  Component 3 - Fan
+    
+    # instantiate
+    fan = SUAVE.Components.Energy.Converters.Fan()   
+    fan.tag                   = 'fan'
+
+    # setup
+    fan.polytropic_efficiency = 0.93
+    fan.pressure_ratio        = 1.7    
+    
+    # add to network
+    ducted_fan.append(fan)
+
+    # ------------------------------------------------------------------
+    #  Component 4 - Fan Nozzle
+    
+    # instantiate
+    nozzle = SUAVE.Components.Energy.Converters.Expansion_Nozzle()   
+    nozzle.tag = 'fan_nozzle'
+
+    # setup
+    nozzle.polytropic_efficiency    = 0.95
+    nozzle.pressure_ratio           = 0.99  
+
+    # add to network
+    ducted_fan.append(nozzle)
+
+    # ------------------------------------------------------------------
+    #Component 5 : thrust (to compute the thrust)
+    thrust = SUAVE.Components.Energy.Processes.Thrust()       
+    thrust.tag                 ='thrust'
+ 
+    #total design thrust (includes all the engines)
+    thrust.total_design        = 2*700. * Units.lbf 
+    
+    
+    # add to network
+    ducted_fan.thrust          = thrust   
+  
+    numerics                   = Data()    
+    eta                        = 1.0
+    
+    #design sizing conditions
+    altitude      = 0.0 * Units.km
+    mach_number   = 0.01
+    isa_deviation = 0.
+    
+    #size the ducted fan
+    ducted_fan_sizing(ducted_fan,mach_number,altitude)
+    
+    
+    battery_ducted_fan                      = SUAVE.Components.Energy.Networks.Serial_Hybrid_Ducted_Fan()
+    battery_ducted_fan.tag                  = 'turbofan'
+    battery_ducted_fan.nacelle_diameter     = ducted_fan.nacelle_diameter
+    battery_ducted_fan.areas                = Data()
+    battery_ducted_fan.areas.wetted         = ducted_fan.areas.wetted
+    battery_ducted_fan.engine_length        = ducted_fan.engine_length
+    battery_ducted_fan.number_of_engines    = ducted_fan.number_of_engines
```

Comment:
Wait is the number of engines multiplied twice? Once for the ducted fan and once for the battery ducted fan?